src/app/views/forgotten-password/forgottenPasswordRequest.jsx

```diff
@@ -34,6 +40,7 @@ const ForgottenPasswordRequest = props => {
                     <button type="submit" className="btn btn--primary margin-top--1">
                         Send the link
                     </button>
+                    {props.waitingResponse ? <div className="form__loader loader loader--dark margin-left--1" /> : ""}
```

Comment:
Maybe its worth replacing this ternary with conditional rendering? something like:
`{props.waitingResponse && <div className="form__loader loader loader--dark margin-left--1" /> }`